src/pages/predictions.vue

```diff
@@ -0,0 +1,416 @@
+    let mainLoaderContainer = document.getElementById("chart-loader");
+    mainLoaderContainer.style.display = "none";
+    this.userFunctions();
+    setTimeout(() => {
+      $("#tab-choose .btn-country").click();
+    }, 500);
+  },
+  data() {
+    return {
+      sortedPredictedArray: this.sortedPredictedArray,
+      PredictionData
+    };
+  },
+  methods: {
+    sortCountriesData: function() {
+      let predictedData = PredictionData;
+      let sortedPredictedArray = [];
+      let totalPredictedCountArray = [];
+      let predictedCountriesArray = Object.keys(predictedData);
+      predictedCountriesArray.forEach(country => {
+        let predictedTimelineArray = Object.values(predictedData[country]);
+        let predictedDateArray = Object.keys(predictedData[country]);
+        predictedDateArray.forEach(e => {
+          moment(e).format("DD-MM-YY");
+        });
+        let predictedActiveCount =
+          predictedTimelineArray[predictedTimelineArray.length - 1].Infected +
+          predictedTimelineArray[predictedTimelineArray.length - 1].Recovered;
+        let latestDate = predictedDateArray[predictedDateArray.length - 1];
+        totalPredictedCountArray.push({
+          country: country,
+          recovered:
+            predictedTimelineArray[predictedTimelineArray.length - 1].Recovered,
+          deaths:
+            predictedTimelineArray[predictedTimelineArray.length - 1].Fatal,
+          confirmed:
+            predictedTimelineArray[predictedTimelineArray.length - 1].Infected +
+            predictedTimelineArray[predictedTimelineArray.length - 1]
+              .Recovered +
+            predictedTimelineArray[predictedTimelineArray.length - 1].Fatal,
+          count: predictedActiveCount,
+          lastUpdated: latestDate
+        });
+      });
+      sortedPredictedArray = totalPredictedCountArray.sort((a, b) =>
+        a.count < b.count ? 1 : b.count < a.count ? -1 : 0
+      );
+      this.sortedPredictedArray = sortedPredictedArray;
+      console.log(sortedPredictedArray);
+      this.generateGraphContent(sortedPredictedArray, predictedData);
+    },
+    generateGraphContent: function(sortedPredictedArray, predictedData) {
+      sortedPredictedArray.forEach((i, index) => {
+        let xlabels = [];
+        let ylabels = [];
+        let dayCount = 0;
+        let recentRecovered = 0;
+        let predictedTimelineArray = Object.values(predictedData[i.country]);
+        predictedTimelineArray.forEach(e => {
+          let confirmed = e.Infected + e.Recovered + e.Fatal;
+          let active = e.Infected + e.Recovered;
+          if (confirmed !== 0) {
+            dayCount = dayCount + 1;
+            ylabels.push(active);
+            xlabels.push(`${dayCount}`);
+          }
+        });
+        this.generateAllCharts(
+          i,
+          index,
+          xlabels,
+          ylabels,
+          sortedPredictedArray
+        );
+      });
+    },
+    generateAllCharts: function(
```

Comment:
remove the active cases count in the predictions graphs and also show the dates on the x-axis instead of the number of days. I also see that the graphs are a bit weird, they are suddenly dropping to 0.